Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -384,6 +384,53 @@ internal static void ActivateHostScene()
             return SendToReady(identity, msg, true, channelId);
         }
 
+        /// <summary>
+        /// Send a message to only clients which are ready
+        /// <para>See <see cref="ClientScene.ready"/></para>
+        /// </summary>
+        /// <typeparam name="T">Message type</typeparam>
+        /// <param name="msg">Message</param>
+        /// <param name="channelId">Transport channel to use</param>
+        /// <returns></returns>
+        public static bool SendToReady<T>(T msg, int channelId) where T : IMessageBase
```

Comment:
```suggestion
        public static bool SendToReady<T>(T msg, int channelId = Channels.DefaultReliable) where T : IMessageBase
```

Use default channel to be consistent with other methods